Detectors/MUON/MCH/Tracking/src/Cluster.cxx

```diff
@@ -0,0 +1,47 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file Cluster.cxx
+/// \brief Implementation of the MCH cluster for internal use
+///
+/// \author Philippe Pillot, Subatech
+
+#include "Cluster.h"
+
+namespace o2
+{
+namespace mch
+{
+
+//_________________________________________________________________________
+Cluster::Cluster(const ClusterStruct& cl) : mX(cl.x), mY(cl.y), mZ(cl.z), mEx(cl.ex), mEy(cl.ey), mUid(cl.uid)
+{
+  /// Constructor from the simple cluster structure
+}
```

Comment:
this looks to me like a default copy constructor. Can be done in the header using `= default`.

Reply:
The copy constructor is already = default in the header. This one is a constructor of an internal Cluster object from a simple clusterStruct.

Reply:
Ok didn't notice this.